config/clusters/nasa-veda/common.values.yaml

```diff
@@ -87,6 +77,36 @@ basehub:
           slug: small
           default: true
           profile_options:
+            image: &image_options
+              choices:
+                display_name: Image
```

Comment:
I don't know why this isn't showing up next to the dropdown like "Node share" does further down in the config

<img width="668" alt="Screenshot 2023-08-17 at 11 56 22" src="https://github.com/2i2c-org/infrastructure/assets/44771837/57ef0794-a47e-4c9b-a102-001ca3cf742d">